src/resources/Comms_zh-CN.properties

```
@@ -0,0 +1,88 @@
+#
+# Copyright (C) 2017-2020 Deltik https://www.deltik.org/
+# Translation modified by Eason Chan / Aobi <eason066@gmail.com> 2020
+#
+# This file is part of SignEdit for Bukkit.
+#
+# SignEdit for Bukkit is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# SignEdit for Bukkit is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with SignEdit for Bukkit.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+# ${pluginName} ${pluginVersion} locale file
+
+# Theme
+reset=§r
+primary=§6
+primaryDark=§7
+primaryLight=§e
+secondary=§f
+highlightBefore=§4
+highlightAfter=§2
+strong=§l
+italic=§o
+strike=§m
+error=§c
+# Display
+prefix={primaryDark}[{primary}{0}{primaryDark}]{reset} {1}
+print_line= {0}＜{1,number}＞{reset} {2}

```

The prepended space here will be ignored.  If you intended for there to be a literal space, it needs to be changed to this:
```suggestion
print_line=\\ {0}＜{1,number}＞{reset} {2}
```